Resources/Prototypes/_Moffstation/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -0,0 +1,47 @@
+- type: entity
+  parent: DrinkGlass
+  id: LemonLimeBittersGlass
+  suffix: lemon lime and bitters
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+        - ReagentId: LemonLimeBitters
+          Quantity: 30
+  - type: Icon
+    sprite: /Textures/_Moffstation/Objects/Consumable/Drinks/lemonlimebitters.rsi
```

```suggestion
    sprite: _Moffstation/Objects/Consumable/Drinks/lemonlimebitters.rsi
```
And similar across the PR.